kotlin-app/src/main/java/com/uber/okbuck/example/MainActivity.kt

```diff
@@ -12,11 +14,16 @@ class MainActivity : DaggerAppCompatActivity() {
     @Inject
     lateinit var analytics: Analytics
 
+    lateinit var smile : ImageView
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         setSupportActionBar(toolbar)
 
+        smile = findViewById(R.id.icon_smile)
```

Comment:
Can you try to reference it from Java class in this module? I think I saw exact same problem but it only happened with Kotlin classes

Reply:
I added the same thing in `app` project (which is java) and it fails the same way.
Trying to find the root cause here